### ISSUE TITLE
install not work with python = 3.8.16

### ISSUE THREAD

**Issue by tankwin08** [showlab/Image2Paragraph#10]
Hi, 
Thanks for the great work. 
try to replicate your work here, i created a new env and try `pip install -r requirements.txt`. but it give me an error. 

ERROR: Could not find a version that satisfies the requirement torch==1.9.0+cu111 (from versions: 1.7.1, 1.8.0, 1.8.1, 1.9.0, 1.9.1, 1.10.0, 1.10.1, 1.10.2, 1.11.0, 1.12.0, 1.12.1, 1.13.0, 1.13.1, 2.0.0)
ERROR: No matching distribution found for torch==1.9.0+cu111

I also try install torch using conda, but it seems not work at the implementation stage. 

Any suggestions are appreciated.

**Comment by FingerRec** [comment]
Hi tankwin08:

Follow previous  install guide https://github.com/showlab/Image2Paragraph/blob/4c7f4436c02f5e9bc878fda0e352aa42995b3e07/install.sh

**Comment by tankwin08** [comment]
Thanks. i did the same thing in previous trial. I tried your instruction, it give me the same error.

<img width="742" alt="image" src="https://user-images.githubusercontent.com/18400035/232363829-a3adcb92-4288-4ca7-b948-eb981724094e.png">

**Comment by FingerRec** [comment]
How about "pip install --upgrade pip".   Because 1.9.0+cu111 is indeed in https://download.pytorch.org/whl/torch_stable.html, as below     
<img width="484" alt="image" src="https://user-images.githubusercontent.com/20349006/232365822-7419590e-2d40-46d5-98ac-0a475b218189.png">

Or what's your system? I only train on linux

**Comment by tankwin08** [comment]
maybe that's the reason, i am using terminal of mac. FYI. update pip did not work.

**Comment by FingerRec** [comment]
Yes, the Mac install is different from linux. Look this page for instruction. https://pytorch.org

**Comment by tankwin08** [comment]
@yahorbarkouski  for only installing torch, you can try `conda install pytorch torchvision torchaudio -c pytorch`, it works for me in python 3.8 with terminal

**Comment by jiangxh18** [comment]
@tankwin08 How did you solve the problem? Can you share it? Thanks!
